### ISSUE TITLE
[Accessibility][Screen Reader-New Trigger] : In scan mode, the state is not announced by the screen reader at New Trigger dialog.

### ISSUE THREAD

**Issue by ashish315** [microsoft/BotFramework-Composer#2081]
**User Experience** 
The AT user might gets confused as state is not announced by the narrator.

**Test Environment**
Browser: Microsoft Edge Canary Version: 82.0.425.0 (Official build)
OS build: 2004 (19564.1005)
Screen Reader: Narrator
URL: [Bot Framework Composer](http://localhost:3000/dialogs/Main)
[**Prerequisite Here**](https://github.com/microsoft/BotFramework-Composer)

**Repro Steps:** 
1. Open the above mentioned URL in EDGE.
2. Select 'Design Flow' from left navigation.
3. Now, navigate to 'New Trigger' button available in the filter dialog pane and activate it..
4. Navigate the dialog in scan mode and observe the issue in "No intent configured for this dialog" dropdown.

**Actual Result:**
Screen reader doesn’t read the state i.e. disabled after the text while announcing in the scan mode.
Ex- Narrator narrates "No intent configured for this dialog" as a plane text
As per narrator narration it appears like plain text.

**Expected:**
The narrator should announce the state as the drop down is disabled.
Ex- It should be announced as "No intent configured for this dialog" disabled text.

**Observation:**
Same issue observed with NVDA.

**MAS Guideline:** [MAS4.2.2](https://microsoft.sharepoint.com/:w:/r/teams/msenable/_layouts/15/WopiFrame.aspx?sourcedoc={216e5b88-a1f8-425c-83e8-11bc4e4d74be})

**Attachment:**
[State should be announced for the diabled field.zip](https://github.com/microsoft/BotFramework-Composer/files/4249867/State.should.be.announced.for.the.diabled.field.zip)

**Comment by liweitian** [comment]
We have changed the UI. The issue does not exist anymore.

**Comment by ashish315** [comment]
As checked in the latest version, the UI has been updated. Due to which this issue doesn't appear to be valid any more. So closing the bug.
